<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Issue74Test" namespace="" name="Issue74Test" abstract="false" final="false" start="2" end="9">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpunit/phpunit/tests/Regression/GitHub/74" file="Issue74Test.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpunit/phpunit/tests/Regression/GitHub/74/Issue74Test.php" size="247" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="3b7ba38228665fe1b4dde2c7a9b47e162e6ffa50" relative="vendor/phpunit/phpunit/tests/Regression/GitHub/74/Issue74Test.php"/>
  <extends full="PHPUnit_Framework_TestCase" namespace="" name="PHPUnit_Framework_TestCase"/>
  <method name="testCreateAndThrowNewExceptionInProcessIsolation" start="4" end="8" abstract="false" final="false" static="false" visibility="public"/>
</class>
